packages/expo-updates/android/src/main/java/expo/modules/updates/selectionpolicy/SelectionPolicyUtils.java

```diff
@@ -0,0 +1,45 @@
+package expo.modules.updates.selectionpolicy;
+
+import android.util.Log;
+
+import org.json.JSONObject;
+
+import java.util.Iterator;
+
+import expo.modules.updates.db.entity.UpdateEntity;
+
+public class SelectionPolicyUtils {
```

Comment:
suggestion: one convention from the Guava library I've liked is to name classes like these using the plural of the the objects they operate on (ex: `Lists` instead of `ListUtils`)

```suggestion
public class SelectionPolicies {
```